### DISCUSSION TITLE
git hub hosting

### DISCUSSION THREAD

**Discussion by 544sourav** [tailwindlabs/tailwindcss#12144] [Help]
i design a site by using tailwind postcss now when i am hosting it on git hub  the tailwind classes are not working  how ever in  local hosting it is perfect.  someone  help me.

**Comment by alex-krasikau** [comment]
Hey @544sourav! Are you able to share the source, so we can check?

It would be helpful, if you send more details about your setup: language, framework.

If you're using `build` locally, make sure to merge build output into your repository, make sure your destination folder is not is `.gitignore` and correctly set a relative path to the CSS file (e.g. `dist/output.css`).

**Comment by 544sourav** [comment]
@alex-krasikau 

first time  i  use postcss and here its repo. link https://github.com/544sourav/Tutor-verse.git
second time i use tailwindcli and here its repo. link https://github.com/544sourav/Tutorverse.git 
but both of them  are not working correctly
